### DISCUSSION TITLE
Manually set up rescanning of specified paths to avoid undetectable files

### DISCUSSION THREAD

**Discussion by skyrim663** [Difegue/LANraragi#859] [Ideas]
LRR Version and OS
LANraragi docker:least ,Ubuntu20.

I am using rclone to mount a network drive. When adding a new archive to the content directory,and if the newly added archive is in a subfolder of the content directory LANraragi cannot detect the file. Even if I manually click the rescan button or the "Restart the file watcher" button.
I found a similar problem here #276 .
And even if the archive is stored in the /content directory, it takes a long time (an hour for me) for LANraragi to scan to the newly added archive.
Perhaps adding the option to set a specific content subdirectory path such as /home/koyomi/lanraragi/content/subfoldername when clicking the "Rescan" button would speed up the scanning process and increase accuracy!

**Comment by Difegue** [comment]
I'd suggest using something else than rclone as FUSE filesystems don't support inotify, which I use for all dynamic filewatching on Linux...  

Triggering manual rescans on specific subdirs sounds like a bad workaround, I wouldn't implement this personally but don't think it'd be that hard either.
